src/routes/tools/extreme-precipitation/_ExploreData.svelte

```diff
@@ -87,20 +91,41 @@
     dataByDate = groupDataByYear(flattenData(data));
   }
 
+  function removeTags(html) {
```

Comment:
It's not too much to worry about but for the instances that this is being used it might just be more straightforward to save them as plain text files instead of markdown or just stash them as JS constants. 

I don't think we need to be super strict about using Markdown files for all content. If it's just a sentence there's not much gain (in terms of reducing the JS bundle size) from putting it in a Markdown file instead of having it as a JS constant.